proto/p4/p4runtime.proto

```diff
@@ -255,10 +255,10 @@ message MeterEntry {
 // meters is not allowed and will return an error. The insertion/deletion
 // should happen as part of INSERT/DELETE on the associated table-entry.
 // For ReadRequest, the scope is defined as follows:
-// - All meter cells for all meters if meter_id = 0 (default).
-// - All meter cells for given meter_id if table_entry.match is empty.
+// - All meter cells for all tables if table_entry is empty (default).
+// - All meter cells of a table if table_entry.table_id is present and
+//   table_entry.match is empty.
```

Comment:
you do not describe the case where `table_entry` is present and `table_entry.table_id == 0`. In the code you seem to treat this case as equivalent to an empty / missing `table_entry` (default), but as I wrote above, I wonder if we should have 2 ways of doing the same thing or if we should mandate that `table_entry` is required.

Reply:
No longer needed. table_id 0 will return an INVALID_ARGUMENT error.